subject_files_status_for_dpdash.py

```diff
@@ -66,7 +67,7 @@ def get_summary_from_phoenix(phoenix_dir: Path) -> pd.DataFrame:
 
     # A/V
     df['interviews'] = df.p.apply(
-        lambda x: _is_file(x, 'interviews', '*.csv'))
+        lambda x: _is_file(x, 'interviews', '*/*.csv'))
```

Comment:
Tashrif's initial idea was to use `glob(recursive=True)` inside `_get_count()` but `Pathlib.path.glob()` does not support `recursive=True` like that of https://docs.python.org/3/library/glob.html#glob.glob